homework-1/css/style.css

```diff
@@ -0,0 +1,1220 @@
+  height: 211px;
+}
+.fixed-top {
+  padding: 0 19px;
+  top: 63px;
+}
+#list-top {
+  height: 40px;
+}
+#banner {
+  height: 211px;
+}
+#banner-submenu {
+  padding: 0 19px;
+  top: 62px;
+}
+#header {
+  padding: 18px 29px;
+}
+#header ul {
+  top: -92px;
+}
+.link-hover-center a {
+  display: inline-block;
+  color: #22BF95;
+  line-height: 1;
+  text-decoration: none;
+  cursor: pointer;
+  position: relative;
+}
+.link-hover-center a:after {
+  display: block;
+  content: "";
+  background-color: #22BF95;
+  height: 1.5px;
+  width: 0%;
+  left: 50%;
+  position: absolute;
+  -webkit-transition: width 0.3s ease-in-out;
+  -moz--transition: width 0.3s ease-in-out;
+  transition: width 0.3s ease-in-out;
+  transform: translateX(-50%);
+}
+.link-hover-center.active a:after {
+  width: 50%;
+}
+.link-hover-center a:hover:after,
+.link-hover-center a:focus:after {
+  width: 50%;
+}
+.tips {
+  padding-top: 52px;
+}
+
+a.tips-active {
+  color: #37e63e;
+}
+a.link-active {
+  color: #37e63e;
+}
+.tips-sub {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.tips-list {
+  position: relative;
+  margin-top: 24px;
+}
+.tips-item-wrap {
+  display: flex;
+  align-items: center;
+}
+.tips-item__num {
+  font-family: "Actor-Regular";
+  font-size: 40px;
+  line-height: 48px;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #E5E5E5;
+  padding-left: 15px;
+  margin-bottom: 8px;
+}
+.tips-item__title {
+  font-family: "Aclonica-Regular";
+  font-size: 16px;
+  line-height: 24px;
+  color: #191919;
+  vertical-align: super;
+  padding-left: 29px;
+  padding-top: 4px;
+}
+.tips__link {
+  display: block;
+  position: relative;
+  width: 335px;
+}
+.tips__link::before, .tips__link::after {
+  content: "";
+  width: 10px;
+  height: 2px;
+  background-color: #37e63e;
+  position: absolute;
+  top: 25px;
+}
+.tips__link::before {
+  transform: rotate(90deg);
+  right: 14px;
+}
+.tips__link::after {
+  transform: rotate(-4deg);
+  right: 14px;
+}
+.tips__link_active::before, .tips__link_active::after {
+  background-color: #37E63e;
+  transition: transform 0.3s ease 0s;
+}
+.tips__link_active::before {
+  transform: rotate(4deg);
+}
+.tips__link_active::after {
+  transform: rotate(0deg);
+}
+.tips__link::before.tips__link_active {
+  transform: rotate(0deg);
+}
+.border-radius {
+  border: 2px solid #37e63e;
+  width: 16px;
+  height: 15px;
+  position: absolute;
+  border-radius: 100%;
+  right: 11px;
+  top: 19px;
+}
+.tips-sub__item-text {
+  width: 335px;
+  font-size: 14px;
+  line-height: 24px;
+  color: #191919;
+}
+.info-item {
+  display: flex;
+  align-items: center;
+  width: 310px;
+  margin-bottom: 24px;
+}
+.border-item {
+  border-top: 1.5px solid #37E63E;
+}
+.border-item-last {
+  border-bottom: 1.5px solid #37E63E;
+}
+.item-title{
+  font-size: 24px;
+  line-height: 32px;
+  margin-right: 115px;
+  margin-top: 18px;
+}
+.img-item {
+  display: flex;
+  justify-content: center;
+}
+.img-item img {
```

Comment:
Do not use tag as selector. Please use specific classes

Reply:
fixed a bug

Reply:
done

Reply:
done